logic/src/jvmMain/kotlin/com/wire/kalium/logic/data/asset/KaliumFileSystem.kt

```diff
@@ -34,4 +43,57 @@ actual class KaliumFileSystem : FileSystem() {
 
     override fun openReadWrite(file: Path, mustCreate: Boolean, mustExist: Boolean): FileHandle =
         SYSTEM.openReadWrite(file, mustCreate, mustExist)
+
+    /**
+     * Provides the root of the cache path, used to store temporary files
+     */
+    actual val rootCachePath: Path = dataStoragePaths.cachePath.value.toPath()
```

Comment:
We need to make sure this directory gets created at a suitable place otherwise the CLI app crashes when it tries to write an asset file.

Reply:
You resolved this but I don't see any code which makes sure the directory exists.

Reply:
It should be pushed now